src/resources/hosts/networks.json

```diff
@@ -131,6 +131,9 @@
     {
       "network": "llnl",
       "files": [
+        {
+          "name": ".host_llnl_pascal.xml.swp"
+        },
```

Comment:
Did you intend to add a host profile for `.host_llnl_pascal.xml.swp`?

Reply:
No I did not. Don't know how I missed that, thanks! Fixed.